telco-hub/configuration/example-overlays-config/acm/options-agentserviceconfig-patch.yaml

```diff
@@ -0,0 +1,37 @@
+# patching mco StorageClass
+
+- op: replace
+  path: /spec/databaseStorage/storageClassName
+  value: "ocs-storagecluster-cephfs" #filesystem StorageClass
+
+- op: replace
+  path: /spec/filesystemStorage/storageClassName
+  value: "ocs-storagecluster-cephfs" #filesystem StorageClass
+
+- op: replace
+  path: /spec/imageStorage/storageClassName
+  value: "ocs-storagecluster-cephfs" #filesystem StorageClass
+
+# comment the following sections if disconnected environment
```

Comment:
Only the "remove" should be commented for disconnected, correct? The replace needs to be updated to point to wherever they host the images in their env.

Reply:
agree, but I could implement yet a good overlay to configure and test the external registry for the spokes. Could we move this to one of these "mini" tasks to do after this PR?. Meanwhile I keep a comment that the remove is temporal (by default) until we implement a good option for that.